_events/OSG-User-School-2023.md

```diff
@@ -0,0 +1,57 @@
+---
+short_title: OSG User School 2023
+title: "OSG User School 2023, Aug. 7–11"
+layout: events
+published: true
+
+excerpt: |
+    Applications for the OSG User School 2023 at the University of Wisconsin-Madison are now open!
+start_date: 2023-08-07
+end_date: 2023-08-11
+location: "University of Wisconsin-Madison"
+link: "https://osg-htc.org/user-school-2023/"
+image: "/images/osg-user-school-2023-event.jpg"
+header_image: "/images/osg-user-school-2023-event.jpg"
+---
+
+{% capture main %}
+
+The OSG User School provides researchers the opportunity to learn how to use high-throughput computing (HTC) systems to run large-scale computing applications at their campus or using the national-scale OSG consortium. Please join us and learn how you can use HTC methods and services to handle large-scale computing applications to advance your research!
```

Comment:
Replace paragraph with:

“
Has your research computing outgrown your available capacity? How could access to **lots** more computing transform your research or that of others? If you have research workloads that can be broken into many independent, parallel computing tasks, we can help.

The OSG User School provides researchers the opportunity to learn how to use high-throughput computing (HTC) systems to run large-scale computing applications at their campus or using the national-scale OSG consortium.
”

[Or feel free to edit, but the existing paragraph has two sentences that say almost the same thing. I stole the other text from our announcement email.]